ui/stories/alert-popup.stories.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable import/extensions */
+import hbs from 'htmlbars-inline-precompile';
+import { storiesOf } from '@storybook/ember';
+import notes from './alert-popup.md';
+import { messageTypes } from '../app/helpers/message-types.js';
+
+storiesOf('AlertPopup/', module)
+  .addParameters({ options: { showPanel: false } })
+  .add(
+    `AlertPopup`,
+    () => ({
+      template: hbs`
+        <h5 class="title is-5">Alert Popup</h5>
+        <AlertPopup
```

Comment:
Can we loop over `messageTypes` here or something similar?

Reply:
yissss!